### DISCUSSION TITLE
朋友们，有些地方我想不太通

### DISCUSSION THREAD

**Discussion by LeeNNN** [WeNeedHome/SummaryOfLoanSuspension#901] [General]
假如，我是说假如，某个政府出面，积极推进某个烂尾楼盘复工。再假如，真像告知书中写的一样，业主集体停贷，直到政府银行开发商把烂尾解决才继续还贷。这中间由于停贷产生的利息（可以说是逾期吗，因为业主这一纸公告不知道有没有法律效果。而且我的印象中逾期的利息很高），真的会由那三方承担吗？政府或者银行真的不会找业主麻烦吗？继续还贷的时候银行要是直接把这加到业主头上怎么办（反正就是他们改改数据库的事情？）？如果到时候硬不还，那法院真能给业主直接拍卖了吧？

**Comment by Accelerator6666** [comment]
你设想一下，如果连日子都过不下去了，还会在乎这些数字吗？

**Comment by b4o** [comment]
法不责众，要看人多少呢，现在已经很多地方政府出台政策了，走一步看一步吧

**Comment by C-Joey** [comment]
先把房子造好吧 能造好就不错了 这么长远的东西 你考虑了也没有用
